command/server.go

```diff
@@ -935,7 +935,8 @@ CLUSTER_SYNTHESIS_COMPLETE:
 		}
 
 		server := &http.Server{
-			Handler: handler,
+			Handler:     handler,
+			IdleTimeout: 10 * time.Minute,
```

Comment:
I tried to check that this is having the desired effect but haven't been successful. Using a build with a very low value (5s) and throwing a long sleep into some operation, the connection wasn't closed. I was even able to `nc 127.0.0.1 8200` and the connection would stay open indefinitely.

I wrote a little Go server to experiment and found that ReadTimeout will close my `nc` test at the right time, but just IdleTimeout won't. I'm not sure what IdleTimeout is really doing. But replacing IdleTimeout with ReadTimeout in Vault didn't close my connection either.